packages/web3-core-helpers/README.md

```diff
@@ -1,31 +1,31 @@
-# web3-core-helpers
+# aion-web3-core-helpers
 
-This is a sub package of [web3.js][repo]
+This is a sub package of [aion_web3][repo].
 
-Helper functions used in [web3.js][repo] packages.
-Please read the [documentation][docs] for more.
+Helper functions used in [aion_web3][repo] packages.
```

Comment:
Do we need to link it twice? I tend to think they're 2 different links when they're right next to each other.